hips/hip-XXXX.md

```diff
@@ -0,0 +1,130 @@
+  ...
+  "oci.example.com": {
+    "auth": "Zm9vOmJhcgo="
+  }
+}
+```
+
+Helm will use the package TBD for updating (and reading) `registries.conf`.
+
+If when reading a registries configation from `registries.conf` results in known configuration Helm doesn't support, Helm must error.
+e.g. `location` or non-empty URI path in `prefix`.
+
+An error reading `registries.conf` must result in an error for the user.
+Otherwise, users who expect configration from `registries.conf` to be effected, will have a unspected fallback.
+
+To account for existing configration in Docker’s registry configuration, Helm will prefer `registries.conf` when resolving OCI registries (including credentials).
+And fall back to the existing store mechanism if `registries.conf` does not contain an entry for the required OCI registry (including if `registries.conf` does not exist).
+
+Helm must expect (and even encourage) users to utilize other tooling to manage `registries.conf`.
+
+## Backwards compatibility
+
+Helm's fallback to Docker's registry configuration ensures the vast majority of existing user workflows remain the same.
+
+However, there are three potential incompatibility scenarios:
+2. A corrupt `registries.conf` will cause an error for existing workflows
+1. An invalid or incompatible with Helm `registries.conf` entry for the given OCI registry will cause the users workflow to fail
+3. Helm’s preference for `registries.conf` will break users who assume credentials are stored in Docker’s registry configuration
+
+The first two can mitigated by users ensuring their systems `registries.conf` is valid, and only includes configuration options Helm supports for the registries they plan to use with Helm.
+
+The last is mitigated by not using `registries.conf` initially unless it exists on the users system.
+
+## Security implications
+
+`registries.conf` introduces a new mechanism for storing OCI credentials, which may introduce credential management vulnerabilities specific to `registries.conf` / `auth.json`
+
+Transitive dependencies of the TBD package for managing registries.conf may introduce security scanner noise (which tends to be a problem in the container library ecosystem)
+
+## How to teach this
+
+Helm's documentation will need to be updated with details of Helm's `registries.conf` support, and fallback to Docker config.
+
+## Reference implementation
+
+<!--
+Link to any existing implementation and details about its state, e.g.
+proof-of-concept.
+-->
+
+## Rejected ideas
+
+### Falling back to Docker’s config upon error reading registries.conf
+
+Falling back upon error means users who expect their configuration to be taken from `registries.conf` will unexpectedly have OCI configuration taken from Docker.
+Potentially resulting in difficult to diagnose failures authenticating to the repository.
+
+Rather than "failing fast" and requring users to ensure their configuration to be valid.
+
+## Open issues
+
+- Support for registies.conf in ORAS
```

Comment:
Since we use ORAS for this, do we need to get agreement from them before this moves forward?

Reply:
Hi @mattfarina ,

Andrew Block brought up this proposal to the ORAS community meeting on Mar 5. There is also a GitHub issue tracked in oras-go https://github.com/oras-project/oras-go/issues/918. 

As this is a dependency for ORAS from Helm v4, what's the expected timeline to get`registies.conf` supported in oras-go?